ex1/index.html

```diff
@@ -3,13 +3,14 @@
 <head>
   <meta charset="UTF-8">
 <link rel="stylesheet" href="main.css">
+<script src="myScript.js"></script>
   <title>The blue button ex</title>
 </head>
 <body>
   <div class="center">
     <h1><u>The blue button</u></h1>
     <h2>This is my blue button</h2>
-    <button type="button" class="button">Hit Me!!!</button>
+    <button type="button" class="button" onclick="myFunction()" id="demo">Hit Me!!!</button>
```

Comment:
Please remove the id and send the function “this”. Like this:
properFunctionNameYoullChoose(this)
See if you could change the styling on “this” in the js

Reply:
It’s important to note that the way you did it is also an option but I’m trying to aim on a clean DOM (read what this is) which has least unnecessary code/attributes.